micrometer-core/build.gradle

```diff
@@ -67,6 +67,8 @@ jar {
 dependencies {
     api project(":micrometer-commons")
     api project(":micrometer-observation")
+    optionalApi 'javax.ws.rs:javax.ws.rs-api' // javax
```

Comment:
I'm confused why this is needed now but it wasn't prior to this pull request. I get a compilation error in `JerseyTags` when I remove this, but that class in micrometer-core is unchanged.